eyelid-match-ops/src/primitives/yashe/conf.rs

```diff
@@ -296,23 +296,15 @@ where
     };
 }
 
-/// Iris bit length polynomial parameters.
-///
-/// This uses the full number of iris bits, which gives an upper bound on benchmarks.
-impl YasheConf for IrisBits {
-    type PolyBN = IrisBitsBN;
-
-    const T: u64 = 2048;
-}
-
 /// Full resolution polynomial parameters.
 ///
 /// These are the parameters for full resolution, according to the Inversed Tech report.
 impl YasheConf for FullRes {
     type PolyBN = FullResBN;
 
-    // VERIFY: max T should be 2^15, not 2^11
-    const T: u64 = 2048;
+    // VERIFY: max T should be 2^15, not 2^12
+    // Larger values cause failures in the positive_multiplication_test().
+    const T: u64 = 4096;
```

Comment:
@emmorais happy to revert this change, but it seems to work with the new dimensions.